### DISCUSSION TITLE
SRS RAN feature list.

### DISCUSSION THREAD

**Discussion by safwan2m** [srsran/srsRAN_Project#511] [User Success Stories]
As part of one of the projects that I am involved in, I was given the task of looking at the feature list published and updating it with additional information after carefully going through the latest SRS RAN release. Please find attached the updated list.

I hope it will be useful to the community and will be very happy to receive your feedback. I will be very happy if you consider this list to be included as part of the published feature list for SRS RAN.
[srsRAN feature list .xlsx](https://github.com/srsran/srsRAN_Project/files/14622981/srsRAN.feature.list.xlsx)

Regards,
Safwan

**Comment by andrepuschmann** [comment]
Thanks for the list.  We maintain a similar document internally but it might make sense to publish it to avoid confusion. We'll get back to that shortly
